mirascope/core/anthropic/call_kwargs.py

```diff
@@ -0,0 +1,13 @@
+"""This module contains the type definition for the Anthropic call keyword arguments."""
+
+from anthropic.types import MessageParam
+
+from mirascope.core.base.call_kwargs import BaseCallKwargs
+
+from .call_params import AnthropicCallParams
+from .tool import AnthropicTool
+
+
+class AnthropicCallKwargs(AnthropicCallParams, BaseCallKwargs[AnthropicTool]):
+    model: str
```

Comment:
what are your thoughts on explicitly marking these as `Required` even though it's redundant (just to be extra clear)?

Reply:
I think it's unnecessary as it would be redundant.
From a type-checking perspective, I believe it's fine without explicitly marking them as Required.

Reply:
Ok, then for consistency throughout the codebase, please remove any other such redundant uses of `Required` (I believe there are only 1 or 2 at most, but should be an easy find/search)

Reply:
I just deleted it.

Reply:
There is another instance in `_CacheControl` in `base.tool`